ludwig/schema/optimizers.py

```diff
@@ -502,15 +502,25 @@ class GradientClippingConfig(schema_utils.BaseMarshmallowConfig):
     """Dataclass that holds gradient clipping parameters."""
```

Comment:
Would you mind removing `desciption = TODO` from `OptimizerDataclassField` above? :D